### ISSUE TITLE
Use system helm instead the packaged one

### ISSUE THREAD

**Issue by ogarcia** [lensapp/lens#2215]
**What would you like to be added**:
A new config entry to set the path to helm binary.

**Why is this needed**:
Because you can provide more tiny application that don't need to include a 47MB binary.

**Environment you are Lens application on:**

- Kubernetes distribution: AKS
- Desktop OS: Arch Linux

**Comment by Nokel81** [comment]
Do you mean a setting similar to the one for kubectl or an option for not bundling helm3 with Lens?

**Comment by ogarcia** [comment]
Now I'm building package without helm using this:
```
make node_modules
make build-extensions
yarn dist:dir
```
But the only method that I found to avoid to have a copy of helm with lens is make a symlink into ` resources/helm3/helm3` of my `/usr/bin/helm`. I think that an option like the one for kubectl is a great solution.

You can see my hack in [Arch Linux pkgbuild](https://aur.archlinux.org/cgit/aur.git/tree/PKGBUILD?h=lens).

**Comment by clain23** [comment]
Using built-in helm on M1 mac results an error:

`/Applications/Lens.app/Contents/Resources/helm3/helm3: Bad CPU type in executable`

Standalone helm works fine.

**Comment by Nokel81** [comment]
@jakolehm Can you investigate the "bad CPU type" issue?

**Comment by ogarcia** [comment]
This is caused because M1 has ARM architecure and in you must package the x86_64 binary in Lens.

**Comment by Nokel81** [comment]
@ogarcia Yes, but in https://github.com/lensapp/lens/pull/3700 shouldn't have introduced it
